readme.txt

```diff
@@ -700,5 +705,7 @@ See [Affiliates Screenshots](http://www.itthinx.com/plugins/affiliates/affiliate
 
 == Upgrade Notice ==
 
-= 2.15.2 =
-* This release contains a fix for the test to find pages containing Affiliates shortcodes.
+= 2.15.3 =
```

Comment:
@eggemplo Rather:

= 2.15.3 =
* This release updates the German and the Spanish translation.